keyboards/tada68/keymaps/dlg/keymap.c

```diff
@@ -0,0 +1,78 @@
+#include QMK_KEYBOARD_H
+
+#define _BL 0
+#define _FL 1
+#define _MAC 2
+
+#define _______ KC_TRNS
```

Comment:
7 space (_______) is already defined to be KC_TRNS in `QMK_KEYBOARD_H`. You don't need this line

```suggestion

```